<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWindow</class>
 <widget class="QDialog" name="DialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMovieName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMovieYear">
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMovieLength"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Director</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMovieDirector"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Top Cast</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMovieCast"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Rating</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMovieRating"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnSelectImage">
       <property name="text">
        <string>Browse Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leMovieImage"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
